README.md

```diff
@@ -217,6 +217,22 @@ to install dependencies and build the code.
 Or you create a new directory under `/home/cpi/apps/Menu` and copy [the file](Packaging/cpi-gamesh/__init__.py) there. After restarting the UI, you can download and compile the game directly from the device itself. See [the readme](Packaging/cpi-gamesh/readme.md) for more details.
 </details>
 
+<details><summary>Amiga via Docker</summary>
+
+### Start the container from the repo root
+```
+docker run -ti --rm -v $(pwd):/work amigadev/crosstools:m68k-amigaos bash
+```
+### Installing dependencies and build
+```
+Packaging/amiga/prep.sh
+cd build

```

Comment:
Let's build in `build-amiga`?

Reply:
Also should `cd build-amiga` *before* running `prep.sh`, otherwise you end up with a bunch of untracket files in the root directory. I'll send this as a PR.